From 915feabe38945799ff8145366a5d7ed79664b8b5 Mon Sep 17 00:00:00 2001
From: Arun Bharadwaj <arun@gumstix.com>
Date: Fri, 2 Oct 2015 13:46:34 -0700
Subject: [PATCH 1/2] Pepper 43C and 43R: Remove redundant pinmux lines

uart0_pins was already defined previously. So remove these
redundant lines.

Signed-off-by: Arun Bharadwaj <arun@gumstix.com>
---
 arch/arm/boot/dts/am335x-pepper-43c.dts | 6 ------
 arch/arm/boot/dts/am335x-pepper-43r.dts | 6 ------
 2 files changed, 12 deletions(-)

diff --git a/arch/arm/boot/dts/am335x-pepper-43c.dts b/arch/arm/boot/dts/am335x-pepper-43c.dts
index 28601a4ecbf5..45b782e37ee6 100644
--- a/arch/arm/boot/dts/am335x-pepper-43c.dts
+++ b/arch/arm/boot/dts/am335x-pepper-43c.dts
@@ -681,12 +681,6 @@
 			0x16C (PIN_INPUT_PULLUP | MUX_MODE3)	/* uart0_rtsn.i2c1_scl */
 		>;
 	};
-	uart0_pins: pinmux_uart0 {
-		pinctrl-single,pins = <
-			0x170 (PIN_INPUT_PULLUP | MUX_MODE0)	/* uart0_rxd.uart0_rxd */
-			0x174 (PIN_OUTPUT_PULLDOWN | MUX_MODE0)	/* uart0_txd.uart0_txd */
-		>;
-	};
 	sd_pins: pinmux_sd_card {
 		pinctrl-single,pins = <
 			0xf0 (PIN_INPUT_PULLUP | MUX_MODE0)	/* mmc0_dat0.mmc0_dat0 */
diff --git a/arch/arm/boot/dts/am335x-pepper-43r.dts b/arch/arm/boot/dts/am335x-pepper-43r.dts
index 55f18a40e002..6b44f95cf169 100644
--- a/arch/arm/boot/dts/am335x-pepper-43r.dts
+++ b/arch/arm/boot/dts/am335x-pepper-43r.dts
@@ -706,12 +706,6 @@
 			0x16C (PIN_INPUT_PULLUP | MUX_MODE3)	/* uart0_rtsn.i2c1_scl */
 		>;
 	};
-	uart0_pins: pinmux_uart0 {
-		pinctrl-single,pins = <
-			0x170 (PIN_INPUT_PULLUP | MUX_MODE0)	/* uart0_rxd.uart0_rxd */
-			0x174 (PIN_OUTPUT_PULLDOWN | MUX_MODE0)	/* uart0_txd.uart0_txd */
-		>;
-	};
 	sd_pins: pinmux_sd_card {
 		pinctrl-single,pins = <
 			0xf0 (PIN_INPUT_PULLUP | MUX_MODE0)	/* mmc0_dat0.mmc0_dat0 */
-- 
2.1.4

